### ISSUE TITLE
css styles

### ISSUE THREAD

**Issue by pbiecek** [ModelOriented/modelDown#52]
Please not that the footer is still grey.
(it's in the class="footer")

New theme for DALEX plots is on GitHub here: https://github.com/pbiecek/DALEX/
So you can also use color codes from
https://github.com/pbiecek/DALEX/blob/master/R/theme_drwhy.R

**Comment by Matiszak** [comment]
@pbiecek I suppose we'd like to have footer normal (white background, not fixed position) - so just removing "footer" class ?

Also, should we give an option to selected the theme in call to `modelDown` ? If so which one should be default ? Or if we're not giving user an option to select theme then which one should we use - theme_mi2 or theme_drwhy ?

**Comment by pbiecek** [comment]
@Matiszak yes, it would be better to remove 'fixed position', IMHO violet will be better than white/grey

For themes, I would stay with the default theme_drwhy, no need for passing this argument
the grey theme_mi2 will not suite current css anyway

**Comment by Matiszak** [comment]
@pbiecek I put PR for violet footer, it looks quiete nice. Although when I tried to use `theme_drwhy` I noticed it is not in the latest `DALEX` release `0.2.7`. When can we expect it to be relased so that we can use it ? Or we can just copy the theme definition into this package so that we can later style the plots independently. But the best option I think would be to use `theme_drwhy` and when needed override the theme. 

![image](https://user-images.githubusercontent.com/20770017/54161584-e4e91b80-4452-11e9-972e-a564e19446f0.png)
